<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-O2O867P" ignored="0" name="Test" tests="12" failures="1" timestamp="2020-11-01T17:51:56 IST" time="24.247" errors="0">
  <testcase name="openURL" time="11.586" classname="testNG.TestNGAnnotations2"/>
  <testcase name="searchProduct" time="0.002" classname="testNG.TestNGAnnotations2"/>
  <testcase name="plp" time="0.002" classname="testNG.TestNGAnnotations2"/>
  <testcase name="pdp" time="0.008" classname="testNG.TestNGAnnotations2">
    <failure type="java.lang.AssertionError" message="Fail">
      <![CDATA[java.lang.AssertionError: Fail
at org.junit.Assert.fail(Assert.java:88)
at testNG.TestNGAnnotations2.pdp(TestNGAnnotations2.java:36)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- pdp -->
  <testcase name="openURL" time="13.0" classname="testNG.TestNGAnnotations"/>
  <testcase name="login" time="0.002" classname="testNG.TestNGAnnotations"/>
  <testcase name="searchProduct" time="0.002" classname="testNG.TestNGAnnotations"/>
  <testcase name="plp" time="0.001" classname="testNG.TestNGAnnotations"/>
  <testcase name="pdp" time="0.001" classname="testNG.TestNGAnnotations"/>
  <testcase name="addToCart" time="0.001" classname="testNG.TestNGAnnotations"/>
  <testcase name="addAddress" time="0.002" classname="testNG.TestNGAnnotations"/>
  <testcase name="paymentGateway" time="0.002" classname="testNG.TestNGAnnotations"/>
</testsuite> <!-- Test -->
